manifests/plugins/redis.pp

```diff
@@ -24,9 +25,13 @@
     content => template('collectd/plugins/redis/redis_info.py.erb'),
   }
 
-  collectd::plugins::plugin_common { 'redis':
-    package_name         => 'collectd-python',
-    plugin_file_name     => '10-redis_master.conf',
-    plugin_template_name => 'redis/10-redis_master.conf.erb',
+  collectd::plugin { 'redis':
+    package_name        => 'collectd-redis',
```

Comment:
This package doesn't exists and should either be removed or changed to collectd-python.  RHEL platforms break on this